community/kernel/src/main/java/org/neo4j/kernel/impl/recovery/RecoveryRequiredChecker.java

```diff
@@ -63,11 +63,12 @@ public boolean isRecoveryRequiredAt( File dataDir ) throws IOException
         }
 
         long logVersion = MetaDataStore.getRecord( pageCache, neoStore, MetaDataStore.Position.LOG_VERSION );
+        MetaDataStore.setRecord( pageCache, neoStore, MetaDataStore.Position.LOG_VERSION, logVersion );
```

Comment:
Why set a new value for that record here? Isn't this a "checker" only?

Reply:
Oops, left over debug code, removing.